pkg/api/handlers/compat/images_build.go

```diff
@@ -130,6 +130,7 @@ func BuildImage(w http.ResponseWriter, r *http.Request) {
 		IDMappingOptions        string   `schema:"idmappingoptions"`
 		IdentityLabel           bool     `schema:"identitylabel"`
 		Ignore                  bool     `schema:"ignore"`
+		InheritLabels           bool     `schema:"InheritLabels"`
```

Comment:
all the other options use all lowercase schema

and this must be at least assigned to the buildoptions one line 696 and below

Reply:
nice catch, done.